.github/scripts/apply_review.py

```
@@ -172,8 +172,10 @@ def _write_file(self, path: str, content: str) -> str:
 
     def _bash(self, cmd: str) -> str:
         env = {k: v for k, v in os.environ.items() if k not in _STRIP_KEYS}
+        import shlex
+        argv = shlex.split(cmd)
         r = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)